### ISSUE TITLE
Question about baseline result

### ISSUE THREAD

**Issue by liyuxuan89** [KMnP/fashionpedia-api#6]
Great repository and dataset. But I have some problem reproducing the result.

I trained a Mask-RCNN with detectron2 without using the attribute value of the dataset. The model was trained with mask_rcnn_R_50_FPN_1x.yaml config of detectron2. I only change the learning rate to 1e-3 for the sake of convergence. It seems that the network can predict the bounding box and masks, but it gives a pretty low confidence. This lead to very low mAP(about 7, should be around 30 or 40). My code works fine on deepfashion2 dataset(mAP 69). 

It will be helpful if you can share some more detailed configurations about how to achieve the baseline.  Thanks !

![visualize2](https://user-images.githubusercontent.com/43963025/87916403-f9591180-caa5-11ea-8841-d6987488efc6.jpg)

![visualize1](https://user-images.githubusercontent.com/43963025/87916371-ef371300-caa5-11ea-850f-0b2b2ccca92d.jpg)

**Comment by richardaecn** [comment]
Hi @liyuxuan89 we used Tensorflow for model training and followed similar settings with Detectron2. Using Detectron2 should be able to reproduce our results.
Currently we are waiting for final reviews of the code to be checked-in. Now we have a landing page and model ckpts / configs released at https://github.com/tensorflow/tpu/tree/master/models/official/detection/projects/fashionpedia . Other code including data conversion, model training and inference will be released soon in the next few days.

**Comment by liyuxuan89** [comment]
@richardaecn Thanks for your reply. I still wonder if the weight for the classification loss needs to be changed. The classification converge very slowly and is 20 times larger than other losses.  I deleted some categories and my model works fine. Also, It will be helpful if you can include the training curve in the released code for reference. 
![image](https://user-images.githubusercontent.com/43963025/88027530-f9bae080-cb69-11ea-909b-fa237fac040b.png)

**Comment by liyuxuan89** [comment]
@richardaecn  Thanks for your reply. I still wonder if the weight for the classification loss needs to be changed. Using the default parameter for MaskRCNN, the classification loss converge very slowly and is 20 times larger than other losses. I deleted some categories and my model works fine. 
![image](https://user-images.githubusercontent.com/43963025/88028186-e6f4db80-cb6a-11ea-83b6-f37765eb2861.png)
left:all categories, output low confidence; right: only some categories, high confidence

**Comment by liyuxuan89** [comment]
@richardaecn Did you use softmax for training classification? The default setting of detectron use softmax. I guess  I should use binary cross entropy for classification like yolov3 since the classes are not mutual exclusive ( A bounding box can include both clothes and accessories).
